sprimber-engine/src/main/java/com/griddynamics/qa/sprimber/lifecycle/allure/AllureSprimber.java

```diff
@@ -166,7 +166,8 @@ private String getHookId(ActionDefinition hookDefinition) {
 
     private String getTestCaseHistoryId(TestCase testCase) {
         try {
-            byte[] bytes = MessageDigest.getInstance("md5").digest(testCase.getName().getBytes(UTF_8));
+            byte[] bytes = MessageDigest.getInstance("md5")
+                    .digest((testCase.getParentName() + testCase.getName()).getBytes(UTF_8));
```

Comment:
Please add more details for unique identification. In current implementation you will get issues in case of similar feature file names